tests/fast-integration/Makefile

```diff
@@ -2,11 +2,13 @@
 ci:
 	npm install
 	npm test
+	npm run test-monitoring
```

Comment:
It would be better to move the monitoring test to the test folder - it would be picked automatically, and you could skip that line.
But you can leave it as it is - I plan a bigger refactoring anyway.

Reply:
I see, then I'd leave it as is since it's going to be reorganized anyway.